### PR TITLE
fix(openapi): parameters can disable openapi

### PR DESCRIPTION
**Pull request by soyuka**

### CODE REVIEW

File: src/OpenApi/Model/PathItem.php [hunk 1]
```diff
@@ -184,7 +184,7 @@ public function withServers(?array $servers = null): self
         return $clone;
     }
 
-    public function withParameters(array $parameters): self
+    public function withParameters(?array $parameters = null): self
```
Comment (bendavies) [inline src/OpenApi/Model/PathItem.php:187]: @soyuka you need to change the return type of `getParameters()` to include `null`

Comment (soyuka) [reply src/OpenApi/Model/PathItem.php:187]: damn I wish I could have phpstan level7 thanks!

Comment (bendavies) [reply src/OpenApi/Model/PathItem.php:187]: Add it! Baseline everything. Fix old code slowly

Comment (soyuka) [reply src/OpenApi/Model/PathItem.php:187]: there's a baseline generator? I think we're close to level 7 anyways, it's just the "document array type" rule that is missing in a lot of files.

Comment (bendavies) [reply src/OpenApi/Model/PathItem.php:187]: `--generate-baseline` flag!

**Pull request merged**